src/accountLinks.js

```diff
@@ -0,0 +1,78 @@
+const { createLink } = require('3id-blockchain-utils')
+const utils = require('./utils')
+
+class AccountLinks {
+  constructor (ceramic, provider) {
+    this._ceramic = ceramic
+    this.provider = provider
+  }
+
+  async create (address, did, proof = null) {
+    const doc = await this._ceramic.createDocument(null, 'account-link', {
+      owners: [await this._convertToCaip10(address)]
+    })
```

Comment:
I guess we can just use the `_getDocId` method here as well 👍

Reply:
Replacing `_getDocId` with `_getDoc` and the other above suggestions is to me a step towards the optimizations discussed in having a document registry (either here or in ceramic). I suggest we think this through more and implement in a followup ticket - I think the value here is in implementing ceramic account links with the minimum of complexity as a first step.

Reply:
I do think the approach I'm suggesting does minimize complexity. Also don't think having a document registry in the ceramic node would change how we would interact with it here.

Reply:
I had started with creating a document without loading it, and it didn't receive updates, which is why I had to use `loadDocument`. Looking into it, this is likely because a document is created in ceramic before checking if one exists in the _docMap object (which is the document registry I'm referring to), and it in turn isn't placed in the registry. This means there are multiple copies of documents, and they are receiving different update events. I'm going to see if I changing this behaviour in ceramic-core fixes this (https://github.com/ceramicnetwork/js-ceramic/blob/develop/packages/ceramic-core/src/ceramic.ts#L68)